app/matplotlib.ipynb

```diff
@@ -86,23 +82,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.xlabel(\"X\")\n",
+    "plt.ylabel(\"Y\")\n",
+    "plt.title(\"Colors\")\n",
+    "plt.show()"
```

Comment:
In Task 2, you need to add a colorbar to the scatter plot to fulfill the requirement of setting the colorbar label. You can use `plt.colorbar()` to add it.